### ISSUE TITLE
Package registry login on Github fails

### ISSUE THREAD

**Issue by mugoh** [rlworkgroup/garage#2171]
I have an open PR and the CI check is unable to build the docker container. It fails with:
![Screenshot from 2020-11-05 15-14-49](https://user-images.githubusercontent.com/44059971/98239381-f1e84600-1f78-11eb-8dd0-7118473791f5.png)

I haven't changed any setup files on the repository and I'm unable to resolve the issue. How should one resolve this?

Thanks

**Comment by ryanjulian** [comment]
@ziyiwu9494

**Comment by krzentner** [comment]
Yeah, unfortunately PR from people outside the `rlworkgroup` organization don't get access to `MJKEY` or `CODEDEV_TOKEN`. We're looking into a solution for this, but in the meantime we can merge and test external PRs manually.

The CI uses multiple jobs, but essentially what it does is equivalent to running `pre-commit` (instructions for which can be found in [CONTRIBUTING.md](https://github.com/rlworkgroup/garage/blob/f856f74eacb0d8f535eb36d9c84309efbc25c384/CONTRIBUTING.md), and `pytest -m "not flaky and not nightly" tests` (after starting an X server, since some tests require a display).
